legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-dbExtension/legend-engine-xt-relationalStore-memsql/legend-engine-xt-relationalStore-memsql-execution/src/main/java/org/finos/legend/engine/authentication/flows/MemSQLStaticWithUserPasswordFlow.java

```diff
@@ -15,20 +15,20 @@
 package org.finos.legend.engine.authentication.flows;
 
 import org.finos.legend.engine.authentication.DatabaseAuthenticationFlow;
+import org.finos.legend.engine.protocol.pure.v1.model.packageableElement.store.relational.connection.specification.MemSqlDatasourceSpecification;
 import org.finos.legend.engine.protocol.pure.v1.model.packageableElement.store.relational.connection.DatabaseType;
 import org.finos.legend.engine.protocol.pure.v1.model.packageableElement.store.relational.connection.authentication.UserNamePasswordAuthenticationStrategy;
-import org.finos.legend.engine.protocol.pure.v1.model.packageableElement.store.relational.connection.specification.StaticDatasourceSpecification;
 import org.finos.legend.engine.shared.core.identity.Credential;
 import org.finos.legend.engine.shared.core.identity.Identity;
 import org.finos.legend.engine.shared.core.identity.credential.PlaintextUserPasswordCredential;
 import org.finos.legend.engine.shared.core.vault.Vault;
 
-public class MemSQLStaticWithUserPasswordFlow implements DatabaseAuthenticationFlow<StaticDatasourceSpecification, UserNamePasswordAuthenticationStrategy>
+public class MemSQLStaticWithUserPasswordFlow implements DatabaseAuthenticationFlow<MemSqlDatasourceSpecification, UserNamePasswordAuthenticationStrategy>
```

Comment:
why do we need to change this?

Reply:
Added a new DatasourceSpecification for MemSQL that has SSL attribute